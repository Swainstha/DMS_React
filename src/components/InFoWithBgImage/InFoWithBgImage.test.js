import testSnapFunction from '../../util/testSnapFunction';
import InfoWithBgImage from './index'
import React from 'react';

testSnapFunction("<InfoWithBgImage/>", "Snapshot test for PhroductGridItem",<InfoWithBgImage /> )