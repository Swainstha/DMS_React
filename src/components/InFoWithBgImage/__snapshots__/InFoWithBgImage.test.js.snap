// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<InfoWithBgImage/> Snapshot test for PhroductGridItem 1`] = `ShallowWrapper {}`;
